### PR TITLE
feat: Convert fair followed artist rail to V3 

### PR DESCRIPTION
**Pull request by rajsam003**
The type of this PR is: **TYPE**

This PR resolves [CX-2281]

### Description

|Before|After|
|---|---|
image|image|

### PR Checklist (tick all before merging)

- [x] I have included screenshots or videos to illustrate my changes, or I have not changed anything that impacts the UI.
- [x] I have tested my changes on **iOS** and **Android**.
- [x] I have added tests/stories for my changes, or my changes don't require testing/stories, or I have included a link to a separate Jira ticket covering the tests.
- [x] I have added a feature flag, or my changes don't require a feature flag. ([How do I add one?](https://github.com/artsy/eigen/blob/main/docs/developing_a_feature.md))
- [x] I have documented any follow-up work that this PR will require, or it does not require any.
- [x] I have added an app state migration, or my changes do not require one. ([What are migrations?](https://github.com/artsy/eigen/blob/main/docs/adding_state_migrations.md))
- [x] I have added a changelog entry below or my changes do not require one.

### To the reviewers 👀

- [ ] I would like at least one of the reviewers to run this PR on the simulator or device.

<details><summary>Changelog updates</summary>

#### Cross-platform user-facing changes

- Convert fair followed artists rail to V3 - sam

#### iOS user-facing changes

-

#### Android user-facing changes

-

#### Dev changes

-

</details>

[CX-2281]: https://artsyproduct.atlassian.net/browse/CX-2281?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Comment by ArtsyOpenSource** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Warnings</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:warning:</td>
      <td>

  ❌ **An error occurred while validating your changelog, please make sure you provided a valid changelog.**
  </td>
    </tr>
  </tbody>
</table>



<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/js">dangerJS</a> against 0708ee26f7ba748e6a0b7f6930467e421448e581
</p>

File: src/lib/Scenes/Fair/Components/FairFollowedArtistsRail.tsx [hunk 1]
```diff
@@ -1,97 +1,50 @@
...
   fair: FairFollowedArtistsRail_fair
 }
 
-export const FairFollowedArtistsRail: React.FC<FairFollowedArtistsRailProps> = ({
-  fair,
-  ...rest
-}) => {
-  if (!fair.followedArtistArtworks?.edges?.length) {
+export const FairFollowedArtistsRail: React.FC<FairFollowedArtistsRailProps> = ({ fair }) => {
+  if (!fair.filterArtworksConnection?.edges?.length) {
```
Comment (olerichter00) [inline src/lib/Scenes/Fair/Components/FairFollowedArtistsRail.tsx:17]: Can we move this below line 21 and change it to

```suggestion
  if (!artworks.length) {
```

File: src/lib/Scenes/Fair/Components/FairFollowedArtistsRail.tsx [hunk 2]
```diff
@@ -1,97 +1,50 @@
...
+        <SectionTitle
+          title="Works by artists you follow"
+          onPress={
+            artworks.length > 2
+              ? () => {
+                  trackEvent(tracks.tappedViewAll(fair))
+                  navigate(`/fair/${fair.slug}/followedArtists`)
+                }
+              : undefined
+          }
```
Comment (olerichter00) [inline src/lib/Scenes/Fair/Components/FairFollowedArtistsRail.tsx:35]: 👍

File: src/lib/Scenes/Fair/Components/FairFollowedArtistsRail.tsx [hunk 3]
```diff
@@ -102,20 +55,43 @@ export const FairFollowedArtistsRailFragmentContainer = createFragmentContainer(
...
+              ...SmallArtworkRail_artworks
             }
           }
         }
       }
     `,
   }
 )
+
+const tracks = {
```
Comment (olerichter00) [inline src/lib/Scenes/Fair/Components/FairFollowedArtistsRail.tsx:70]: 🥇

**Review by olerichter00 — APPROVED** [review]
Great work! Just added one comment, feel free to merge after it is resolved.

**Comment by olerichter00** [comment]
#squashongreen

**Pull request merged**